app/helpers/application_helper.rb

```diff
@@ -77,6 +77,20 @@ def get_large_dropzone_id(location, reply_to)
     end
   end
 
+  # used in views/editor/_toolbar.html.erb
```

Comment:
thanks for the comment here, these can be tough to trace! There is the ability to make view-specific helpers like `editor_helper.rb` but often if they only contain a few lines, it can make them harder to find!